magento/pwa-studio
# #2209 Display category description (with PageBuilder support as well) on the category view

## Description
Display the category description on the category page after the category name.
The category description renderer supports the PageBuilder content.

## Related Issue
Closes #2209 .

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to the category page that has a description in the admin area.
2. Verify the category description shows up.

## Screenshots / Screen Captures (if appropriate)

## Checklist
